flow-typed/Comment.js

```diff
@@ -221,10 +220,20 @@ declare type ModerationAmIParams = {
 };
 
 declare type SettingsParams = {
-  channel_name: string,
+  channel_name?: string,
   channel_id: string,
-  signature: string,
-  signing_ts: string,
+  signature?: string,
+  signing_ts?: string,
+};
+
+declare type SettingsResponse = {
+  words?: string,
+  comments_enabled: boolean,
+  min_tip_amount_comment: number,
+  min_tip_amount_super_chat: number,
```

Comment:
Note that Commentron seem to enforce a minimum amount of `0.01`.  Not sure if this is good or bad.
Setting to anything smaller like `0.001` will automatically reset it to `0`.  It would be nicer if it clamped to `0.01` instead.

Reply:
I think a min of 0.01 is fine.